week-4/implement/count.test.js

```diff
@@ -1,5 +1,24 @@
 // implement a function countChar that counts the number of times a character occurs in a string
 
+function countChar(str, char) {
+  let index = 0;
```

Comment:
Can you think of a more descriptive name for this variable? We already have `i` which keeps track of the index